webknossos/webknossos/dataset/layer.py

```diff
@@ -634,21 +634,30 @@ def _assert_mag_does_not_exist_yet(
                 )
             )
 
+    def _get_dataset_from_align_with_other_layers(
+        self, align_with_other_layers: Union[bool, "Dataset"]
+    ) -> Optional["Dataset"]:
+        if type(align_with_other_layers) is bool:
+            return self.dataset if align_with_other_layers else None
+        else:
+            return align_with_other_layers
+
     def downsample(
         self,
         from_mag: Optional[Mag] = None,
-        max_mag: Optional[Mag] = None,
+        coarsest_mag: Optional[Mag] = None,
```

Comment:
Please add this to the changelog, as this might break keyword arguments (no fallback with the old name necessary IMO).